python_backend/api/v1/endpoints/surveys.py

```
@@ -41,18 +42,25 @@ class SubmitRequest(BaseModel):
-            survey_answers=req.answers,
-            store=store,
-            store_vector=True
-        )
-        return res
+        # Temporarily return mock data
+        return {
+            "id": 1,
+            "userId": req.user_id,
+            "baseFormalityLevel": 75,
+            "baseFriendlinessLevel": 60,
+            "baseEmotionLevel": 50,
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)